### PR TITLE
Qt/Debugger: Implement "JIT" widget

### PR DESCRIPTION
**Pull request by spycrab**

### CODE REVIEW

File: Source/Core/DolphinQt2/MenuBar.cpp [hunk 1]
```diff
@@ -102,6 +105,20 @@ void MenuBar::OnEmulationStateChanged(Core::State state)
...
+  m_jit_disable_cache->setEnabled(!running);
+  m_jit_clear_cache->setEnabled(running);
+  m_jit_log_coverage->setEnabled(!running);
+  m_jit_search_instruction->setEnabled(running);
+
+  for (QAction* action :
+       {m_jit_off, m_jit_loadstore_off, m_jit_loadstore_lbzx_off, m_jit_loadstore_lxz_off,
+        m_jit_loadstore_lwz_off, m_jit_loadstore_floating_off, m_jit_loadstore_paired_off,
+        m_jit_floatingpoint_off, m_jit_integer_off, m_jit_paired_off, m_jit_systemregisters_off})
+    action->setEnabled(running);
```
Comment (BhaaLseN) [inline Source/Core/DolphinQt2/MenuBar.cpp]: Some braces perhaps, since it spans over multiple lines?

File: Source/Core/DolphinQt2/MenuBar.cpp [hunk 2]
```diff
@@ -1041,3 +1171,37 @@ void MenuBar::PatchHLEFunctions()
...
+
+void MenuBar::LogInstructions()
+{
+  PPCTables::LogCompiledInstructions();
+}
+
+void MenuBar::SearchInstruction()
+{
+  bool good;
+  QString op = QInputDialog::getText(this, tr("Change instruction"), tr("New instruction:"),
```
Comment (BhaaLseN) [inline Source/Core/DolphinQt2/MenuBar.cpp]: What does "Search" have to do with "Change" instruction?
Can ignore if thats the same Wx does (unless someone has a better idea on how to do this).

File: Source/Core/DolphinQt2/MenuBar.cpp [hunk 3]
```diff
@@ -1041,3 +1171,37 @@ void MenuBar::PatchHLEFunctions()
...
+                                     QLineEdit::Normal, QStringLiteral(""), &good);
+
+  if (!good)
+    return;
+
+  bool found = false;
+  for (u32 addr = 0x80000000; addr < 0x80180000; addr += 4)
```
Comment (leoetlino) [inline Source/Core/DolphinQt2/MenuBar.cpp]: `0x80180000` should be `0x81800000`.

Comment (BhaaLseN) [reply Source/Core/DolphinQt2/MenuBar.cpp]: That just means its already wrong in Wx :P

https://github.com/dolphin-emu/dolphin/blob/a9987588eb4fbd69a6fb9f405a83827084b1a874/Source/Core/DolphinWX/Debugger/CodeWindow.cpp#L553

**Pull request merged**